creator-node/nginx_conf/nginx.conf

```diff
@@ -46,10 +47,12 @@ http {
             proxy_cache_use_stale error timeout updating http_500 http_502 http_503 http_504;
             proxy_cache_background_update on;
 
-            # Cache only responses with status code = 200 for some duration before considered stale.
+            # Cache responses with certain status codes for some duration before considered stale.
             # Stale implies content will be fetched from the upstream server. Stale content WILL NOT
             # BE REMOVED from the cache.
             proxy_cache_valid 200 12h;
+            proxy_cache_valid 401 403 0s;
+            proxy_cache_valid any 1s;
```

Comment:
the any 1s after doesn't overwrite the former? (just checking)

Reply:
correct, it does not (from [their docs](http://nginx.org/en/docs/http/ngx_http_proxy_module.html#proxy_cache_valid))
<img width="557" alt="Screen Shot 2022-09-20 at 5 27 48 PM" src="https://user-images.githubusercontent.com/3323835/191367986-b590a41a-fdc9-43ac-9861-eb57cdd718fd.png">